components/gcp-click-to-deploy/src/DeployForm.tsx

```diff
@@ -25,6 +26,9 @@ import clusterJinjaPath from './configs/cluster.jinja';
 // selects auto domain then we should automatically supply the suffix
 // <hostname>.endpoints.<Project>.cloud.goog
 
+// Assume user access app via kubectl proxy
+const BackendAddress = 'http://127.0.0.1:8001/api/v1/namespaces/default/services/kubeflow-controller:8080/proxy/';
```

Comment:
Why do we need to make this assumption? Can we parameterize this based on how the user accessed the click to deploy app?

What if we had our go backend serve the click to deploy app? i.e. can we just add a handler to our go server that returns the javascript click to deploy app?

Reply:
Will make it a parameter in yaml spec.

I think having go backend serve the app won't help with resolving backend api address: backend itself won't know from which address will user access it unless configured.